pywebostv/controls.py

```diff
@@ -195,6 +202,7 @@ def list_audio_output_sources(self):
         "get_audio_output": {
```

Comment:
Just curious, this is just `subscribe_get_audio_output(..)`, but do other subscribe responses from the TV need `subscription_validation`?

Reply:
I will check for all subscriptions and report back :)

Reply:
These are the responses from the rest of subscriptions (3) with respect to the original code:

- `ssap://audio/getVolume`: *works*
```python
{'type': 'response', 'id': '...', 'payload': {'returnValue': True, 'volumeStatus': {'volumeLimitable': True, 'activeStatus': True, 'maxVolume': 100, 'volumeLimiter': 'none', 'soundOutput': 'tv_speaker', 'volume': 0, 'cause': 'volumeDown', 'mode': 'normal', 'externalDeviceControl': False, 'muteStatus': False, 'volumeSyncable': True, 'adjustVolume': True}, 'callerId': 'com.webos.platformstarfish'}}
```
- `ssap://tv/getCurrentChannel`: **fails**
```python
{'type': 'response', 'id': '...', 'payload': {'subscribed': True, 'channelTypeName': 'Terrestrial Digital TV', 'channelNumber': '5', 'isChannelChanged': False, 'channelName': 'Telecinco HD', 'isInteractiveRestrictionChannel': False, 'isSkipped': False, 'isReplaceChannel': False, 'isLocked': False, 'isHEVCChannel': False, 'channelModeId': 0, 'isInvisible': False, 'isScrambled': False, 'signalChannelId': '16_190_8916', 'physicalNumber': 34, 'hybridtvType': 'HBBTV', 'isDescrambled': False, 'channelModeName': 'Terrestrial', 'channelId': '1_34_5_0_16_190_8916', 'favoriteGroup': None, 'channelTypeId': 1, 'isFineTuned': False, 'dualChannel': {'dualChannelTypeName': None, 'dualChannelNumber': None, 'dualChannelId': None, 'dualChannelTypeId': 255}}}
```
- `ssap://com.webos.applicationManager/getForegroundAppInfo`: *works*
```python
{'type': 'response', 'id': '...', 'payload': {'returnValue': True, 'appId': 'com.webos.app.livetv', 'processId': '', 'windowId': ''}}
```

It's strange that half the subscriptions never worked and nobody noticed.
Either way, it should be fixed, you tell me if we fix it in this PR/issue or make a new one ;)

*PS: I checked the api version is still 1 for the tv endpoint*

Reply:
Good catch, thanks a lot for your hard work here -- really appreciate it! :)

BTW, I'm not sure if anyone uses TV channels anymore these days -- probably why it went unnoticed for years.

As for your question about PR, totally up to you. I'm fine either way.